make/istio/run

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+: "${GIT_ROOT:=$(git rev-parse --show-toplevel)}"
+source "${GIT_ROOT}/bin/common/versions.sh"
+source "${GIT_ROOT}/make/include/defaults"
+
+# Download and install the Istio release.
+istio_output_dir="${GIT_ROOT}/output/istio"
+istio_path="${istio_output_dir}/${ISTIO_VERSION}"
+if [ ! -d "$istio_path" ]; then
+  rm -rf "${istio_output_dir:?}/*"
+  mkdir -p "${istio_path}"
+  curl -L "https://github.com/istio/istio/releases/download/${ISTIO_VERSION}/istio-${ISTIO_VERSION}-linux.tar.gz" \
+    | tar zx -C "${istio_path}" --strip-components=1
+fi
+
+
+chart="${istio_path}/install/kubernetes/helm/istio"
+
+# Install the Istio Helm chart.
+args=(
+  "${ISTIO_HELM_RELEASE}"
+  --namespace "${ISTIO_NAMESPACE}"
+  --timeout 1800
+)
+
+values=(
+  --set "gateways.istio-ingressgateway.type=NodePort"
+  --set "gateways.istio-egressgateway.enabled=false"
+  --set "tracing.enabled=true"
+  --set "grafana.enabled=true"
+)
+
+helm upgrade --install "${args[@]}" "${chart}" "${values[@]}"
+
+path_with_istio="PATH=\"${istio_path}/bin:\${PATH}\""
+printf "Done. You might want to 'export %s' to get Istio binaries onto your \$PATH.\\n" "${path_with_istio}"
```

Comment:
Maybe add it in [`.envrc`](https://github.com/SUSE/scf/blob/340b8e6c511a59394110d2ba5409da12dc3f23e1/.envrc#L71) too? (With the version number in there)